alphagov/paas-cf
# WIP: [#154770854] Enable the Redis service for everyone.

## What

We want to make this generally available. This enables access to the
broker for all tenants on the platform.

With this being generally available, there's no longer a need to
explicitly enable access for the tests.

## How to review

* Deploy to a dev env
* Verify that the redis service is available in all orgs

## Who can review

Not me.
